Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleGeoTrackingSceneView.swift

```diff
@@ -186,17 +204,32 @@ public struct WorldScaleGeoTrackingSceneView: View {
         // GPS location is not accurate, we won't end up below the earth's surface.
         let altitude = (location.position.z ?? 0) + location.verticalAccuracy
         
-        cameraController.originCamera = Camera(
-            latitude: location.position.y,
-            longitude: location.position.x,
-            altitude: altitude,
-            heading: calibrationHeading ?? 0,
-            pitch: 90,
-            roll: 0
-        )
+        if !initialCameraIsSet {
+            let heading = 0.0
+            viewModel.cameraController.originCamera = Camera(
+                latitude: location.position.y,
+                longitude: location.position.x,
+                altitude: altitude,
+                heading: heading,
+                pitch: 90,
+                roll: 0
+            )
+            viewModel.calibrationElevation = altitude
+        } else {
+            // Ignore location updates when calibrating heading and elevation.
+            guard !viewModel.isCalibrating else { return }
+            viewModel.cameraController.originCamera = Camera(
+                latitude: location.position.y,
+                longitude: location.position.x,
+                altitude: viewModel.calibrationElevation ?? altitude,
+                heading: viewModel.calibrationHeading ?? 0,
+                pitch: 90,
+                roll: 0
+            )
+        }
```

Comment:
You can keep the old code here to first try using calibration values, and fall back to default values. I'll change these in #574